### ISSUE TITLE
Npm install not working because of requirement 

### ISSUE THREAD

**Issue by StoneyEagle** [microlinkhq/youtube-dl-exec#3]
Error: Cannot find module 'fs/promises'
Require stack:
-<folder>/node_modules/youtube-dl-exec/scripts/postinstall.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (<folder>/node_modules/youtube-dl-exec/scripts/postinstall.js:4:12)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '<folder>/node_modules/youtube-dl-exec/scripts/postinstall.js'
  ]
}

This is probably fixed with changing 
```js 
const fs = require("fs/promises");
``` 
to 
```js 
const fs = require("fs").promises;
```

I reverted back to version 1.0.0 for now.
Love the package 😉

**Comment by Kikobeats** [comment]
wow that strange

<img width="374" alt="Screen Shot 2021-03-04 at 16 07 57" src="https://user-images.githubusercontent.com/2096101/109984165-cc3f3380-7d03-11eb-9b77-f93166c2a870.png">

what node version are you using?

**Comment by Kikobeats** [comment]
hmm looks like it's an API not available at node 10:

<img width="541" alt="Screen Shot 2021-03-04 at 16 09 35" src="https://user-images.githubusercontent.com/2096101/109984457-158f8300-7d04-11eb-8cfd-1e5e91cd2df8.png">

let me do the change!

**Comment by StoneyEagle** [comment]
I'm using node v12.20.0.
I will upgrade to 14+

**Comment by Kikobeats** [comment]
shipped at `youtube-dl-exec@1.1.1`, can you try it and confirm it works fine for you?
